edith-step/src/UserChart.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { mount, unmount } from 'enzyme';
+import ReceiptInput from './ReceiptInput';
```

Comment:
ReceiptInput not used, import can be deleted

Reply:
Done.